addons/transitions/Transitions.gd

```diff
@@ -5,16 +5,28 @@ signal pre_transition
 const FadeScene = preload("res://addons/transitions/FadeScene.tscn")
 var _root:Viewport
 var _current_scene = null
+var scene_container:Node setget _set_scene_container
```

Comment:
Could you rename this to `_scene_container`?

Reply:
I'm more than happy to change it to `_scene_container`. It *is* a public variable though, with a setter that updates the `_current_scene` variable. Should public variables still have the underscore prefix? (Obviously your project and I will accept whatever coding conventions you're going for)

Reply:
Sorry, my bad, you're right. It shouldn't have a prefix.